RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 -----
 * [*] Site Creation: Fixed layout of domain input field for RTL languages. [#18006]
 * [*] [internal] The FAB (blue button to create posts/stories/pages) creation/life cycle was changed [#18026]
+* [*] Stats: Remove Customise and Grow Audience cards from Stats Insights [#18051]
```

Comment:
We should update this to reflect that we are only removing Customize. Or better we can omit this line, since #18072 already has an entry that describes a re-organization of the Insights screen